python/pylibcudf/tests/test_utils.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2025 NVIDIA CORPORATION.
+import pyarrow as pa
+import pytest
+from utils import assert_column_eq
+
+import pylibcudf as plc
+
+
+@pytest.mark.parametrize(
+    "values",
+    [
+        [1, 2, 3],
+        [[1, 2], [3]],
```

Comment:
If you need to add another commit, could you add a test with one of the values being `None`?